common/determined_common/api/request.py

```diff
@@ -11,14 +11,15 @@
 import determined_common.requests
 from determined_common.api import authentication, errors
 
-# The path to a file containing an SSL certificate to trust specifically for the master, if any.
+# The path to a file containing an SSL certificate to trust specifically for the master, if any, or
+# False to disable cert verification entirely.
```

Comment:
What is this new feature for?  Or rather, why did we used to be ok without it but as part of this change now we need it?

Reply:
I'm fine with it as-is, I was just curious.